src/2019/ExtensibilityTemplatePack2019.csproj

```diff
@@ -211,6 +217,12 @@
       <HintPath>..\..\packages\TemplateBuilder.1.1.6.1\lib\TemplateBuilder.dll</HintPath>
     </Reference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This should be `<content>`, right?

Reply:
Thanks for pointing out.  Good learning experience.